### PR TITLE
Feat(FWS-62): Ability to edit fitness service package

### PR DESCRIPTION
**Pull request by Abdur-Siddiqui**
JIRA: link to jira ticket
https://champlainsaintlambert.atlassian.net/browse/FWS-62

What is the ticket about and why are we making this change?

This ticket concerns being able to update a fitness service as a personal trainer in both English and French. As a personal trainer, I want to be able to edit fitness package details such as changing the price or service name and much more.

What are the various changes and what other modules do those changes affect?

-integrated testing 
-Unit Testing
-backend for the update
-Front end for the update
-Localization in both French and english

UI Change:

Before 

image

Now

image
image

### CODE REVIEW

File: fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        const { name, value } = e.target;
+        setFitnessPackage(prevState => ({
+            ...prevState,
+            [name]: value
+        }));
+    };
+
+    const handleUpdate = (serviceId) => {
+         // Replace with actual service ID
+        fetch(`http://localhost:8080/api/v1/fitnessPackages/${serviceId}`, {
```
Comment (Eli293) [inline fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js:48]: should this not use BASE URL for the deployed website?

File: fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            method: "PUT",
+            headers: {
+                "Authorization": "Bearer " + accessToken,
+                "Content-Type": "application/json",
+            },
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js:53]: Forgot to change to base url

**Review by AlexeiTimbro — APPROVED** [review]
Good job Abdur keep it up

File: fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+                }
+                return response.json();
+            })
+            .then((data) => {
+                console.log("Fitness package updated successfully", data);
+                // Handle successful update (e.g., show confirmation, refresh data)
+            })
+            .catch((error) => {
+                console.error("Error updating fitness package:", error);
+            });
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js:68]: Why are you not setting any data when you get by id

File: fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+                    >
+                        Update Package
+                    </Button>
+                </Form>
+            </Container>
+
+            <FooterNotLoggedIn/>
+        </div>
+    );
+}
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js:131]: forgot localization

File: fitwsarah_frontend/src/views/HomePage/Home.js [hunk 5]
```diff
@@ -127,137 +188,227 @@ function Home() {
...
+                        <Button variant="primary" onClick={() => handleUpdateService(selectedService.serviceId)}>
+                            Update Service
+                        </Button>
+                    </Form>
+                </Modal.Body>
+            </Modal>
+
+
+            <FooterNotLoggedIn/>
+        </div>
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/views/HomePage/Home.js:409]: You forgot localization

**Review by emilegirardGit — APPROVED** [review]
Fix last comments and its all good

**Review by Eli293 — APPROVED** [review]
Looks good.

**Review by emilegirardGit — APPROVED** [review]

File: fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js (hunk 1 above)
Comment (Abdur-Siddiqui) [reply fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js:48]: I ended up not using this file.

**Pull request merged**